packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useMemo, useState } from 'react';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { useModal } from '../../../../../components/ModalProvider';
+import DocumentsIcon from '../../../../../public/images/ic-documents.svg';
+import IdCardIcon from '../../../../../public/images/ic-id-card.svg';
+import SelfieIcon from '../../../../../public/images/ic-selfie.svg';
+import { useDynamicLeverageModalState } from '../../../components/DynamicLeverageContext';
+import { getJurisdictionContents } from '../jurisdiction-contents/jurisdiction-contents';
+import { TJurisdictionCardItems, TJurisdictionCardSection } from '../jurisdiction-contents/props.types';
+import JurisdictionCardBack from './JurisdictionCardBack';
+import JurisdictionCardRow from './JurisdictionCardRow';
+import JurisdictionCardTag from './JurisdictionCardTag';
+import JurisdictionCardVerificationTag from './JurisdictionCardVerificationTag';
+
+type TJurisdictionCardProps = {
+    isAdded: boolean;
+    isSelected: boolean;
+    jurisdiction: string;
+    onSelect: (clickedJurisdiction: string) => void;
+    tag?: string;
+};
+
+type TVerificationDocumentsMapper = {
+    [key: string]: {
+        category: 'poa' | 'poi' | null;
+        icon?: JSX.Element;
+    };
+};
+
+const verificationDocumentsMapper: TVerificationDocumentsMapper = {
+    documentNumber: {
+        category: 'poi',
+        icon: <IdCardIcon />,
+    },
+    nameAndAddress: {
+        category: 'poa',
+        icon: <DocumentsIcon />,
+    },
+    notApplicable: {
```

Comment:
I think we had a [card](https://app.clickup.com/t/20696747/CFDS-1392) to remove the minus (-) sign icon and `not_applicable` status before...